src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -109,8 +105,12 @@ const GroupTabDetailsWrapper = ({
           )}
         </PageSection>
       </Tab>
-      <Tab eventKey={1} title="Group info" aria-label="Group info tab">
-        {activeTabKey === 1 && ( // helps to lazy load the component
+      <Tab
+        eventKey={groupTabKeys.groupInfo}
+        title="Group info"
+        aria-label="Group info tab"
+      >
+        {groupTabKey === groupTabKeys.groupInfo && ( // helps to lazy load the component
```

Comment:
> // helps to lazy load the component

How?

Reply:
Though you are right, this is another set of Tabs 
We need to save the filters only for the "Hybrid Inventory Tabs" (the inner Tabs component), according to the ticket

In case we need to also keep the filtering when switching to the "Group Info" tab, the fix would probably be what you mentioned here. But, at the moment, it was not mentioned on the ticket